ojdbc-provider-jackson-oson/src/main/java/oracle/jdbc/provider/oson/OsonGenerator.java

```diff
@@ -103,12 +104,12 @@ protected OsonGenerator(int features, ObjectCodec codec, OracleJsonGenerator gen
 
   /**
    * Flushes the generator and writes any buffered output to the underlying stream.
```

Comment:
Indeed but... that is a no-op because what' been called here (i.e gen.close()) actually does nothing. 
Shall we add the comment here or on doc of the underlying method ?